monkey/common/types.py

```diff
@@ -18,6 +18,18 @@
 MachineID: TypeAlias = PositiveInt
 
 
+class NetworkPort(ConstrainedInt):
```

Comment:
Out of curiosity, does mypy play well with `ConstrainedInt`? I know `conint` and the like cause issues.

Reply:
I'm not seeing any compliants. See https://github.com/pydantic/pydantic/issues/239#issuecomment-409644889 for more details.